core-feature-pack/src/main/resources/content/bin/common.ps1

```diff
@@ -0,0 +1,256 @@
+if($PSVersionTable.PSVersion.Major -lt 2) {
+    Write-Warning "posh-git requires PowerShell 2.0 or better; you have version $($Host.Version)."
+    return
+}
+
+$SCRIPTS_HOME = (Get-ChildItem $MyInvocation.MyCommand.Path).Directory.FullName
+$RESOLVED_JBOSS_HOME = (Get-ChildItem $MyInvocation.MyCommand.Path).Directory.Parent.FullName
+
+
+$global:SECMGR = $false
+$global:DEBUG_MODE=$false
+$global:DEBUG_PORT=8787
+$global:RUN_IN_BACKGROUND=$false
+#module opts that are passed to jboss modules
+$global:MODULE_OPTS = @()
+
+# A collection of functions that are used by the other scripts
+
+Function Set-Env {
+  $key = $args[0]
+  $value = $args[1]
+  Set-Content -Path env:$key -Value $value
+}
+
+Function Get-Env {
+  $key = $args[0]
+  if( Test-Path env:$key ) {
+    return (Get-ChildItem env:$key).Value
+  }
+  return $args[1]
+}
+
+Function Get-String {
+  $value = ''
+  foreach($k in $args) {
+    $value += $k
+  }
+  return $value
+}
+
+Function String-To-Array($value) {
+  $res = @('')
+  if (-Not(Test-Path $value)){
+  	return $res
+  }
+  $tmpArr = $value.split()
+  
+  foreach ($str in $tmpArr) {
+    if ($str -ne '') {
+	  $res += $str
+	}
+  }
+  return $res
+}
+
+Function Display-Environment { 
+
+# Display our environment
+echo "================================================================================="
+echo ""
+echo "  WildFly Bootstrap Environment"
+echo ""
+echo "  JBOSS_HOME: $JBOSS_HOME"
+echo ""
+echo "  JAVA: $JAVA"
+echo ""
+echo "  MODULE_OPTS: $MODULE_OPTS"
+echo ""
+echo "  JAVA_OPTS: $JAVA_OPTS"
+echo ""
+echo "================================================================================="
+echo ""
+
+}
+
+Function Get-Java-Arguments {
+Param(
+   [Parameter(Mandatory=$true)]
+   [string]$entryModule,
+   [string]$logFileProperties = "$JBOSS_CONFIG_DIR/logging.properties",
+   [string]$logFile = "$JBOSS_LOG_DIR/server.log",
```

Comment:
This doesn't seem like a good default. `server.log` should only be used for standalone servers.